rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/get_list_hvac_systems_associated_with_zone.py

```diff
@@ -0,0 +1,34 @@
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+
+
+def get_list_hvac_systems_associated_with_zone(rmi, zone_id):
+    """
+     Get the list of the heating ventilaiton and cooling system ids associated with a zone in either the U_RMD, P_RMD, or B_RMD.
+
+    Parameters
+    ----------
+    rmi: dict RMI at RuleSetModelInstance level
+    zone_id: str Zone id
+
+    Returns: list  A list that saves all the HVAC systems associated with the zone.
+    -------
+
+    """
+    return sorted(
```

Comment:
The RDS does not require a sort. My guess is this sorted is used for testing purpose is that correct?